compiler/rustc_parse/src/parser/mod.rs

```diff
@@ -583,6 +583,20 @@ impl<'a> Parser<'a> {
         self.token.is_keyword(kw)
     }
 
+    fn check_keyword_case(&mut self, kw: Symbol, case_insensitive: bool) -> bool {
+        if self.check_keyword(kw) {
+            return true;
+        }
+
+        if case_insensitive
+        && let Some((ident, /* is_raw */ false)) = self.token.ident()
+        && ident.as_str().to_lowercase() == kw.as_str().to_lowercase() {
```

Comment:
Surprised rustfix didn't catch this:
```suggestion
        if case_insensitive
            && let Some((ident, /* is_raw */ false)) = self.token.ident()
            && ident.as_str().to_lowercase() == kw.as_str().to_lowercase()
        {
```

Reply:
Did you mean rustfmt? It doesn't work at all with if let chains currently

Reply:
We also don't really have a style guide for it, so :shrug: